content/end.html

```diff
@@ -68,53 +68,30 @@ <h3> UWaterloo Software Engineering </h3>
 
 <h3> Inspirations </h3>
 <p class="full">
-Systems Design Engineering 2017 Class Profile
-<a target="_blank" href="http://joeyloi.com/SYDE2017classprofile.pdf">
-http://joeyloi.com/SYDE2017classprofile.pdf
-</a>
-</p>
-
-<p class="full">
-HackerRank 2018 Developer Skills Report
-<a target="_blank" href="http://research.hackerrank.com/developer-skills/2018/">
-http://research.hackerrank.com/developer-skills/2018/
+Software Engineering 2018 Class Profile
+<a target="_blank" href="https://classprofile.andyzhang.net/">
+  https://classprofile.andyzhang.net/
 </a>
 </p>
 
 <p class="full">
-StackOverflow 2018 Developer Survey Results
-<a target="_blank" href="https://insights.stackoverflow.com/survey/2018/">
-https://insights.stackoverflow.com/survey/2018/
+Electrical and Computer Engineering 2021 Class Profile
```

Comment:
I think we should maybe reference other class profiles or at least the oldest one per program